scripts/run_module_image.sh

```diff
@@ -36,12 +36,30 @@ skopeo copy ${TLS_OPTIONS} docker://${IMAGE_NAME} dir:${TARGET_DIRECTORY}
 
 FILENAME=$(cat ${TARGET_DIRECTORY}/manifest.json  | jq -c '.layers[] | select(.mediaType=="application/gzip").digest[7:]' | tr -d \")
 
-echo -e "\n--- Extracting resources from file:" ${FILENAME}
+# if no application/gzip files found that means that we have a raw Kubernetes YAML file and we need to identify it before apply
+if [ -z "${FILENAME}" ]
+then
+  POTENTIAL_YAML_FILES=$(cat ${TARGET_DIRECTORY}/manifest.json  | jq -c '.layers[] | select(.mediaType=="application/octet-stream").digest[7:]' | tr -d \")
+  POTENTIAL_YAMLS_ARRAY=($POTENTIAL_YAML_FILES)
+
+  for RAW_K8S_YAML in "${POTENTIAL_YAMLS_ARRAY[@]}"
+  do
+    KIND_VALUE=$(cat ${TARGET_DIRECTORY}/${RAW_K8S_YAML} | yq e '.kind' -)    
+    if echo "$KIND_VALUE" | grep -q "CustomResourceDefinition"; 
+    then
+      echo -e "\n--- Installing BTP Manager in ${NAMESPACE} namespace"
```

Comment:
Please augment message with information about whether we install using helm or kubectl apply